base/deprecated.jl

```diff
@@ -1569,6 +1569,10 @@ end
 
 @deprecate String(io::GenericIOBuffer) String(take!(copy(io)))
 
+@eval Base begin
```

Comment:
this file is in Base? can set the extra boolean argument to false so it's not exported